Disable inclusion of the standard libpath into linker flags/libs.
This patch is Solaris linker specific and is not suitable for community.

--- a/libidn2.pc.in	2017-11-03 03:06:55.308324817 +0000
+++ b/libidn2.pc.in	2017-11-03 03:07:35.573026800 +0000
@@ -7,5 +7,5 @@
 Description: Library implementing IDNA2008 and TR46
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}
-Libs: -L${libdir} -lidn2
+Libs: -lidn2
 Libs.private: @LTLIBICONV@ @LTLIBUNISTRING@
